test/generators/extra_pages_generator_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ExtraPagesGeneratorTest < Rails::Generators::TestCase
+  tests MountainView::Generators::ExtraPagesGenerator
+
+  destination File.expand_path("../../tmp2", __FILE__)
```

Comment:
Thanks! Not sure if it has to do with the failing test. Lets see what happens after changing it, and we can investigate if it presists.